packages/app/src/store/studio-store.ts

```diff
@@ -113,8 +113,25 @@ interface StudioRecorderState {
   newTestLineNumber?: number
 }
 
+function getUrlParams () {
+  const url = new URL(window.location.href)
+  const hashParams = new URLSearchParams(url.hash)
+
+  const testId = hashParams.get('testId')
+  const suiteId = hashParams.get('suiteId')
+  const visitUrl = hashParams.get('url')
+  const newTestLineNumber = hashParams.get('newTestLineNumber') ? Number(hashParams.get('newTestLineNumber')) : undefined
+  const cloudStudioSessionId = hashParams.get('cloudStudioSessionId')
```

Comment:
Since we are already in the context of `studio`, suggest renaming the url parameter `sessionId`.